app/src/docs/manageAccess.md

```diff
@@ -0,0 +1,156 @@
+| --------------- | ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `user.manage`   | Complete administrative access | • Manage users and roles<br>• Assign permissions to users and roles<br>• Manage read permissions on packets and packet groups<br>• Automatic read access to all packets |
+| `packet.run`    | Execute packet groups          | • Access packet runner interface<br>• Run any packet groups<br>• View packet run logs                                                                                   |
+| `outpack.read`  | Read outpack server data       | • Make read calls to the outpack server                                                                                                                                 |
+| `outpack.write` | Write outpack server data      | • Make write calls to the outpack server<br>• Includes `outpack.read` capabilities                                                                                      |
+
+### Scoped Permissions
+
+These permissions can be applied globally or restricted to specific resources using scope qualifiers.
+
+#### `packet.read`
+
+Grants read access to packets with flexible scoping options.
+
+**Scope Examples:**
+
+- `packet.read` - Global read access to all packets
+- `packet.read:packetGroup:group1` - Read access to all packets in group1
+- `packet.read:packet:group1:id` - Read access to specific packet with ID "id" in group1
+- `packet.read:tag:tag2` *(coming soon)* - Read access to packets tagged with "tag2"
+
+#### `packet.manage`
+
+Allows management of read access permissions for packets.
+
+**Capabilities:**
+
+- Grant read access to users or roles for packet groups or individual packets
+- Revoke read access from users or roles
+- Automatically includes `packet.read` for managed resources
+
+**Scope Examples:**
+
+- `packet.manage` - Can manage read access for any packet
+- `packet.manage:packetGroup:group1` - Can manage read access for group1 packets only
+- `packet.manage:packet:group1:id` - Can manage read access for one specific packet
+- `packet.manage:tag:tag2` *(coming soon)* - Can manage read access for tagged packets
+
+### Permission Dependencies
+
+Some permissions automatically include others:
+
+- `outpack.write` → includes `outpack.read`
+- `user.manage` → includes global `packet.read`
+- `packet.manage` → includes `packet.read` for the same scope
+
+## Managing Access
+
+### Creating Roles
+
+1. Navigate to **Manage Roles**
+2. Click **Add Role**
+3. Enter role name and description
+4. Select appropriate global permissions
+5. Click **Save**
+6. Assign users and scoped permissions as needed
+
+### Updating Role Assignments
```

Comment:
There's a slight disconnect between the implementation detail language used above and the user-friendly "how to navigate the UI" language here. Perhaps there's some technical details that can be split out of the first part? Not a big deal and it might be worth getting a vibe check from someone on the research side

Reply:
yeah maybe il just remove the navifate the UI part.... as thats pretty intuitive as is? 

or maybe il just leave and get feedback from research first

Reply:
definitely keep it, but perhaps move it earlier so that a less technical reader finds what they want before assuming the docs are not for them?